helm/csi-isilon/templates/controller.yaml

```diff
@@ -196,7 +198,7 @@ spec:
             - "--v=5"
             - "--leader-election"
             - "--enable-node-watcher=true"
-            - "--monitor-interval={{ .Values.volumeHealthMonitorInterval | default "60s" }}"
+            - "--monitor-interval={{ .Values.healthMonitor.volumeHealthMonitorInterval | default "60s" }}"
```

Comment:
should it be .values.controller.healthmonitor?

Reply:
correct. Fixed this in the new commit.